src/SpotifyApi.NetCore/IFollowApi.cs

```diff
@@ -1,73 +1,73 @@
         /// <summary>
-        /// Check if Current User Follows Artists
+        /// Check if Current User Follows Artists or Users
         /// </summary>
-        /// <param name="ids">Required. A comma-separated list of the artists Spotify IDs to check. A maximum of 50 IDs can be sent in one request.</param>
+        /// <param name="artistIds">Required. A comma-separated list of the artist Spotify IDs to check. A maximum of artist 50 IDs can be sent in one request.</param>
         /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
         /// <remarks>
         /// https://developer.spotify.com/documentation/web-api/reference/follow/check-current-user-follows/
         /// </remarks>
         Task<bool[]> CheckCurrentUserFollowsArtists(
-            string[] ids,
+            string[] artistIds,
             string accessToken = null
             );
 
         /// <summary>
-        /// Check if Current User Follows Artists
+        /// Check if Current User Follows Artists or Users
         /// </summary>
-        /// <param name="ids">Required. A comma-separated list of the artists Spotify IDs to check. A maximum of 50 IDs can be sent in one request.</param>
+        /// <param name="artistIds">Required. A comma-separated list of the artist Spotify IDs. A maximum of 50 artist IDs can be sent in one request. A minimum of 1 artist id is required.</param>
         /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
         /// <remarks>
         /// https://developer.spotify.com/documentation/web-api/reference/follow/check-current-user-follows/
         /// </remarks>
         Task<T> CheckCurrentUserFollowsArtists<T>(
-            string[] ids,
+            string[] artistIds,
             string accessToken = null
             );
 
         /// <summary>
-        /// Check if Current User Follows Users
+        /// Check if Current User Follows Artists or Users
         /// </summary>
-        /// <param name="ids">Required. A comma-separated list of the users Spotify IDs to check. A maximum of 50 IDs can be sent in one request.</param>
+        /// <param name="userIds">Required. A comma-separated list of the user Spotify IDs. A maximum of 50 user IDs can be sent in one request. A minimum of 1 user id is required.</param>
         /// <returns>bool[] an array of true or false values, in the same order in which the ids were specified.</returns>
         /// <remarks>
         /// https://developer.spotify.com/documentation/web-api/reference/follow/check-current-user-follows/
         /// </remarks>
         Task<bool[]> CheckCurrentUserFollowsUsers(
-            string[] ids,
+            string[] userIds,
             string accessToken = null
             );
 
         /// <summary>
-        /// Check if Current User Follows Users
+        /// Check if Current User Follows Artists or Users
```

Comment:
"Check to see if the current user is following one or more other Spotify users."